### PR TITLE
remove every trace of config

### PR DESCRIPTION
**Pull request by ainsleymcgrath**
ah yes, how young i was. so naive. i wrote the concept for the project
in my beautiful bullet journal. "High config," i said, as i attempted to
articulate the Guiding Principles of my Sweet Baby™.

the years passed; the decades and centuries flew by. clouds of
starlings, wingbeats synchrnoized across oceans and continents. in
valhalla, i meet Salomon. "Wait, what are you config-ing again"

I reply. "idk. lol"

### CODE REVIEW

**Comment by SalomonSmeke** [comment]
💋 :zen: 🐍

File: nothing_cli/localization/en.json [hunk 1]
```diff
@@ -31,11 +31,12 @@
   "home_dot_nothing_exists_warn": "There is already a .nothing dir in your home",
   "cwd_dot_nothing_exists_warn": "There is already a .nothing dir in your cwd",
   "made_cwd_dot_nothing_dir": "Created ./.nothing/",
-  "made_home_dot_nothing_dir": "Created ~/.nothing",
+  "made_sample_procedure": "Created a sample Procdure called 'nothing' for you in {directory} 💖",
   "wrote_config": "Wrote ~/.nothing/_config.yaml",
```
Comment (ainsleymcgrath) [inline nothing_cli/localization/en.json:35]: delete this

File: nothing_cli/main.py [hunk 2]
```diff
@@ -114,18 +107,15 @@ def new(
...
+        True, "--edit-after", "-A", help=glot["new_edit_after_option_help"]
     ),
     overwrite: bool = typer.Option(
         False, "--overwrite", "-O", help=glot["new_overwrite_option_help"]
     ),
 ):
     """Subcommand for creating new Procedures"""
 
-    destination_dir = HOME_DOT_NOTHING_DIR if not local else CWD_DOT_NOTHING_DIR
+    destination_dir = HOME_DOT_NOTHING_DIR if not global_ else CWD_DOT_NOTHING_DIR
```
Comment (ainsleymcgrath) [inline nothing_cli/main.py:118]: backwards logic 🤦

File: nothing_cli/models.py [hunk 3]
```diff
@@ -82,30 +79,23 @@ class Procedure(BaseModel):
...
+    title = glot["default_title"]
+    description = glot["default_description"]
+    steps: List[Step] = steps_from_yaml_block(glot["default_steps"])
     context: List[ContextItem] = context_items_from_yaml_list(
-        config.default_context_list
+        [{glot["default_context_name_name"]: glot["default_context_name_prompt"]}]
     )
 
 
 class ProcedureCreateExpert(Procedure):
```
Comment (ainsleymcgrath) [inline nothing_cli/models.py:96]: mark this guy 4 deprecation

**Pull request merged**